### DISCUSSION TITLE
treshold color

### DISCUSSION THREAD

**Discussion by Brancus** [grafana/grafana#44646] [Dashboards, Panels & Visualizations]
When using the Threshold from a query, the color scheme is not customizable and is always set to green below threshold and red above threshold. (Gauge visualization)
It should be possible to customize this and set red below threshold and green above, for example.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
